lib/active_reporting/reporting_dimension.rb

```diff
@@ -188,7 +188,12 @@ def datetime_drill_label_fragment(column)
     end
 
     def datetime_drill_postgress(column)
-      "DATE_TRUNC('#{@datetime_drill}', #{column})"
+      case @datetime_drill.to_sym
+      when :date
+        "DATE('#{column})"
```

Comment:
I think you meant to have a single quote on the other side of the variable here?

Reply:
My bad! Fixed.